src/Search.tsx

```diff
@@ -1,12 +1,20 @@
 import debounce from "lodash.debounce";
-import { useEffect, useCallback, useState } from "react";
+import { useEffect, useCallback, useState, ChangeEvent } from "react";
 import { OBJ_URL, SEARCH_URL } from "./consts";
 import { Art } from "./models/Art";
 import Image from "./Image";
+type TArtworkIds = {
```

Comment:
I've also seen some people use T suffix.. although I don't think its really needed at all

Reply:
yeah i normally don't really like it but i was having a namespace clash with the Art model. i actually think my type naming here in general could be rethought, but i figure we'll get there when we get there...